runOpenVpn.sh

```diff
@@ -5,14 +5,37 @@ then
 	if [ -f /etc/openvpn/"${OPEN_VPN_CONFIG}".ovpn ]
   	then
 		echo "Starting OpenVPN using config ${OPEN_VPN_CONFIG}.ovpn"
-		exec openvpn --config /etc/openvpn/"${OPEN_VPN_CONFIG}".ovpn
+		OPEN_VPN_CONFIG=/etc/openvpn/${OPEN_VPN_CONFIG}.ovpn
 	else
 		echo "Supplied config ${OPEN_VPN_CONFIG}.ovpn could not be found."
 		echo "Using default OpenVPN gateway: Netherlands"
-		exec openvpn --config /etc/openvpn/Netherlands.ovpn
+		OPEN_VPN_CONFIG=/etc/openvpn/Netherlands.ovpn
 	fi
 else
 	echo "No VPN configuration provided. Using default: Netherlands"
-	exec openvpn --config /etc/openvpn/Netherlands.ovpn
+	OPEN_VPN_CONFIG=/etc/openvpn/Netherlands.ovpn
 fi
 
+# override resolv.conf
+if [ "$RESOLV_OVERRIDE" != "**None**" ];
+then
+  echo "Overriding resolv.conf..."
+  printf "$RESOLV_OVERRIDE" > /etc/resolv.conf
+fi
+
+# add PIA user/pass
+if [ "$PIA_USERNAME" != "**None**" ];
+then
+  echo "Setting PIA credentials..."
+  mkdir -p /config
+  echo $PIA_USERNAME > /config/pia-credentials.txt
+  echo $PIA_PASSWORD >> /config/pia-credentials.txt
+else
+  echo "Not setting PIA credentials."
+fi
+
+dockerize \
+  -template /etc/transmission-daemon/settings.json:/etc/transmission-daemon/settings.json \
```

Comment:
Setting transmission-daemon setting from environment variables. Then, if the port updater should when TRANSMISSION_RPC_AUTHENTICATION_REQUIRED is set to true, the updater should also read the environment variables